eng/common/templates/job/job.yml

```diff
@@ -192,11 +192,13 @@ jobs:
       displayName: Publish Test Results
       inputs:
         testResultsFormat: 'xUnit'
-        testResultsFiles: '*.xml' 
+        testResultsFiles: '*.xml'
         searchFolder: '$(Build.SourcesDirectory)/artifacts/TestResults/$(_BuildConfig)'
+        testRunTitle: $(AgentOsName)-$(BuildConfiguration)-xunit
+        mergeTestResults: true
```

Comment:
I know this is in `eng/common` which is not to be touched. I won't merge this, just testing something and will contribute to Arcade if it yields a good result.